src/DropdownButton/DropdownButton.tsx

```diff
@@ -27,16 +26,25 @@ export interface DropdownButtonProps extends DefaultProps {
   align?: "left" | "right"
 }
 
-const BaseDropdownButton = styled(Button)<{ isOpen: boolean; textColor: string }>(({ isOpen, textColor, theme }) => ({
+const BaseDropdownButton = styled(Button)<{ isOpen: boolean }>(({ isOpen, theme }) => ({
   paddingRight: 0,
-  ...(isOpen && { boxShadow: `0 0 0 1px ${expandColor(theme, textColor)} inset` }),
+  marginRight: 0,
+  ...(isOpen && {
+    borderBottom: "1px solid #fff",
```

Comment:
This would come back to bite us in the end when we want to implement the dark mode unless we change it like so:

```suggestion
    borderBottom: `1px solid ${({theme}) => theme.color.white}`,
```